NeoDebuggerUI/ViewModels/MainWindowViewModel.cs

```diff
@@ -157,8 +162,8 @@ public async Task OpenRunDialog()
 			var modalWindow = new InvokeWindow();
 			var task = modalWindow.ShowDialog();
 			await Task.Run(()=> task.Wait());
-		}
-
 
+            ConsumedGas = "GAS used: " + string.Format("{0:N4}", DebuggerStore.instance.manager.UsedGasCost);
```

Comment:
Can you place "Gas used: " in the xml layout? And maybe we would not need to set this value here if you use a getter, as you did in the last PR